src/main/java/com/odysseusinc/arachne/datanode/controller/BaseDataSourceController.java

```diff
@@ -240,9 +238,7 @@ public JsonResult unpublishAndDeleteOnCentral(Long dataSourceId) {
     )
     public List<OptionDTO> getDBMSTypes() {
 
-        return Arrays.stream(DBMSType.values())
-                .map(dbms -> conversionService.convert(dbms, OptionDTO.class))
-                .collect(Collectors.toList());
+        return integrationService.getDbmsTypes();
```

Comment:
What was the purpose, reasoning of the change?

Reply:
copy paste